scipy/stats/_stats_py.py

```diff
@@ -4363,6 +4366,10 @@ def confidence_interval(self, confidence_level=0.95, method=None):
         return ci
 
 
+xp_capabilities_table['pearsonr'] = dict(cpu_only=True)
+
+
+@xp_capabilities(**xp_capabilities_table['pearsonr'])
```

Comment:
Assuming we want to leave `skip_xp_backends` right now (I don't want to touch it, anyway), I'm thinking this can accept a sequence of dictionaries, each of which defines valid keargs input to `skip_xp_backends`. We process these to produce the capabilities table here, and it would then be very easy to chain together a test decorator from them since each element is valid input to `skip_xp_backends`.